charts/eks/templates/daemonset-hostpath-mapper.yaml

```diff
@@ -47,7 +47,7 @@ spec:
         {{- if .Values.hostpathMapper.image }}
         image: "{{ .Values.defaultImageRegistry }}{{ .Values.hostpathMapper.image }}"
         {{- else }}
-        image: "{{ .Values.defaultImageRegistry }}loftsh/vcluster:{{ .Chart.Version }}"
+        image: "{{ .Values.defaultImageRegistry }}ghcr.io/loft-sh/vcluster:{{ .Chart.Version }}"
```

Comment:
I am not sure this will work correctly when `{{ .Values.defaultImageRegistry }}` is set :thinking: 
When it is set, the `ghcr.io/` should probably be omitted. 
And when the `{{ .Values.defaultImageRegistry }}` is set, should this be backward compatible - "loftsh" org without a dash? @FabianKramm wdyt?

Reply:
Yeah, was initially confused by this as well, but the image `ghcr.io/loft-sh/vcluster ` when used with `push-images.sh --registry ghcr.io/rohantmp` will be pushed to `ghcr.io/rohantmp/ghcr.io/loft-sh/vcluster` and that is by design.